specification/maps/data-plane/Render/preview/1.0/render.json

```diff
@@ -165,7 +165,8 @@
-        "operationId": "GetMapStaticImage",
+        "operationId": "Render_GetMapImage",
+        "x-ms-client_name": "GetMapStaticImage",
```

Comment:
@tjprescott  "x-ms-client_name" is an invalid name, you should use "x-ms-client-name" instead.

Reply:
@alankashiwa

Reply:
@kairu-ms @tjprescott Thanks for pointing this out. I've opened a PR to fix the typo: https://github.com/Azure/azure-rest-api-specs/pull/17290